docs/source/conf.py

```diff
@@ -98,7 +102,7 @@
 todo_include_todos = True
 
 # Mock imports that won't and don't have to be installed in ReadTheDocs environment
-autodoc_mock_imports = ["boto3", "botocore"]
+autodoc_mock_imports = ["boto3", "botocore", "pyproj", "pytest"]
```

Comment:
Do we still have to include `pyproj` here or has that issue been resolved?

Reply:
both of these can be removed